### PR TITLE
MB-5223 no_MtoID_no_error

### PR DESCRIPTION
**Pull request by NamibiaLT**
## Description

When a move has not been created for an order, and that order is updated, it is possible for a nil uuid to be stored in place of the MtoID. No notification will be sent to the prime and no error will be returned for that nil MtoID

## Reviewer Notes

Please check my tests are testing what is expected.

## Setup

To run the tests in event_test.go:
Happy path test: `go test ./pkg/services/event --testify.m TestMoveOrderEventTrigger -v`
Unhappy path to test when there is a nil MtoID`go test ./pkg/services/event --testify.m TestNotificationEventHandler -v`

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-5223) for this change

### CODE REVIEW

File: pkg/services/event/event_test.go [hunk 1]
```diff
@@ -369,3 +369,40 @@ func (suite *EventServiceSuite) TestMoveOrderEventTrigger() {
...
+			Request:         &dummyRequest,
+			EndpointKey:     InternalUpdateOrdersEndpointKey,
+			HandlerContext:  handler,
+			DBConnection:    suite.DB(),
+		}
+		_, err := TriggerEvent(event)
+		suite.NoError(err)
+
+		afterCount, _ := suite.DB().Count(&models.WebhookNotification{})
+		suite.Equal(count, afterCount)
```
Comment (abbyoung) [inline pkg/services/event/event_test.go:401]: nice!

File: pkg/services/event/event_test.go [hunk 2]
```diff
@@ -369,3 +369,40 @@ func (suite *EventServiceSuite) TestMoveOrderEventTrigger() {
...
+
+		// No notificationn adn nil error returned
```
Comment (abbyoung) [inline pkg/services/event/event_test.go:403]: a couple small typos in the comment 😄

File: pkg/services/event/notification_test.go [hunk 3]
```diff
@@ -201,3 +201,5 @@ func (suite *EventServiceSuite) TestAssembleMoveOrderPayload() {
 		}
 	})
 }
+
+// Check Success that if thre is an MTO, a notification is STORED, but if there isn't no error is returned.
```
Comment (abbyoung) [inline pkg/services/event/notification_test.go:205]: small typo in comment (*there)

**Review by abbyoung — APPROVED** [review]
looks good! a couple very small typos in comments, but code looks great.

File: pkg/services/event/event_test.go (hunk 2 above)
Comment (NamibiaLT) [reply pkg/services/event/event_test.go:403]: Good catch! Going to make those changes now.

**Review by NamibiaLT — COMMENTED** [review]
Just updated the files based on the feedback that was received.

**Pull request merged**